src/test/java/data/TaskManagerTest.java

```diff
@@ -56,7 +56,7 @@ void updateTodo_validInput_updatesTask() throws TaskManagerException {
         addTask(date, initialTaskDescription, testTaskType, dummyTestDates);
 
         // Act
-        updateTask(date, 0, updatedTaskDescription);
+        //updateTask(date, 0, updatedTaskDescription);
 
         // Assert
         assertEquals(updatedTaskDescription, taskManager.getTasksForDate(date).get(0).getName());
```

Comment:
This test seems to be failing. Could you check what changes you made that is causing it to fail?

Reply:
On closer look, line 59 is commented out which may be the reason why the task description is not being updated